MekHQ/src/mekhq/gui/dialog/BombsDialog.java

```diff
@@ -97,6 +122,23 @@ private JPanel buildButtonPanel() {
     public void actionPerformed(ActionEvent e) {
         if (okayButton.equals(e.getSource())) {
             bombPanel.applyChoice();
+            newLoadout = bombPanel.getChoice();
+            
+            //Get difference between starting bomb load and new bomb load
+            for (int type = 0; type < BombType.B_NUM; type++) {
+                if(bombChoices[type] != newLoadout[type]) {
+                    newLoadout[type] = bombChoices[type] - newLoadout[type];
+                } else newLoadout[type] = 0;

```

Comment:
We use the "One True Brace" style (1TBS) that includes braces around control statement bodies even if they are only a single statement.